packages/shared/ReactFeatureFlags.js

```diff
@@ -210,7 +221,7 @@ export const enableRenderableContext = __NEXT_MAJOR__;
 // when we plan to enable them.
 // -----------------------------------------------------------------------------
 
-export const disableModulePatternComponents = __NEXT_MAJOR__;
+export const disableModulePatternComponents = true;
```

Comment:
This is under chopping block?

Reply:
I have a PR to remove it entirely https://github.com/facebook/react/pull/27742